conf/messages.en

```diff
@@ -510,6 +510,14 @@ clientSecondContactPhone.error.invalid = Telephone number can only include numbe
 clientSecondContactPhone.error.length = The first contact’s telephone number must be 24 characters or less
 clientSecondContactPhone.change.hidden = client second contact telephone number
 
+clientFirstContactHavePhone.title = Can we call {0} by telephone?
```

Comment:
there shouldn't be a parameter in the title, check the prototype but I think it's probably "Can we call them by telephone?"